docker/hisat2/make_grch38_snp_tran_ensembl.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/sh
+
+#
+# Downloads sequence for the GRCh38 release N version of H. sapiens (human) from
+# Ensembl.
+#
+# Note that Ensembl's GRCh38 build has three categories of compressed fasta
+# files:
+#
+# The base files, named ??.fa.gz
+#
+# By default, this script builds and index for just the base files,
+# since alignments to those sequences are the most useful.  To change
+# which categories are built by this script, edit the CHRS_TO_INDEX
+# variable below.
+#
+
+ENSEMBL_RELEASE=$1
```

Comment:
Is this a Hisat script that's been copied over? I would love to see some more documentation in the script if you wrote it, or if not, a quick tl;dr on how to use it.

Reply:
it comes with HISAT2 source code